config/application.php

```diff
@@ -116,21 +116,6 @@
 // Limit the number of post revisions that Wordpress stores (true (default WP): store every revision)
 Config::define('WP_POST_REVISIONS', env('WP_POST_REVISIONS') ?: true);
 
-/**
```

Comment:
How are these changes related to the linting configuration?

Reply:
The change in this file was prompted by CodeClimate when I pushed the configuration as part of this work. This was the code block that got moved by both us and Pantheon, and it looks like when we resolved their changes we ended up with two of them. CodeClimate caught the duplication.

We didn't specifically talk last week about whether to use the duplication check on CodeClimate or not, but it has been part of how we use the platform in the past, and given how CodeClimate behaves, it seems like a good tool to use selectively (rather than on GitHub, which is global).

Reply:
Got it thanks.

For future consideration, noting this removal as a side effect of implementing code climate would have helped me as a reviewer understand wtf was going on. (No change is requested now as we've now talked about it so I understand the removal).